CHANGELOG.md

```diff
@@ -34,6 +34,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 -->
 
+## [1.2.6]
+Added gas type in Contract AbiItem
+
```

Comment:
Please have a closer look at this file again! 😄

Reply:
@maektwain Apologies, could you change this so it's only a single entry in the existing `1.2.7` section of the changelog, at the bottom of the file?

Reply:
Made this change via Github UI, resolving....